Releases/update-versions.py

```diff
@@ -174,6 +174,27 @@ def UpdateTags(version_data, firebase_version, first=False):
     LogOrRun("git push origin '{}'".format(tag))
 
 
+def CheckVersions(version_data):
+  """Ensure that versions do not already exist as tags.
+
+  Args:
+    version_data: dictionary of versions to be updated.
+  """
+  error = False
+  for pod, version in version_data.items():
+    name = pod[len('Firebase'):]
```

Comment:
Currently all pods start with "Firebase" but this might not be always the case in the future. Should we at least add a check and throw an error if pod doesn't start with "Firebase"?

Reply:
Good catch - especially with the new `Google` pods coming in M53. I added support for Google pods and verified that all pods are prefixed with `Firebase` or `Google`.

cc: @mikehaney24